homeassistant/components/asuswrt.py

```diff
@@ -0,0 +1,62 @@
+"""
+Support for the asuswrt component.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/asuswrt/
+"""
+import logging
+
+import voluptuous as vol
+
+from homeassistant.const import (
+    CONF_HOST, CONF_PASSWORD, CONF_USERNAME, CONF_PORT, CONF_MODE,
+    CONF_PROTOCOL)
+
+from homeassistant.helpers import config_validation as cv
+from homeassistant.helpers.discovery import load_platform
+
+REQUIREMENTS = ['aioasuswrt==1.1.0']
+
+_LOGGER = logging.getLogger(__name__)
+
+DOMAIN = "asuswrt"
+DATA_ASUSWRT = 'ASUSWRT'
```

Comment:
A pattern I've seen several components use is just setting this equal to `DOMAIN`

Reply:
Agreed :)